src/CourseDashboard.cpp

```diff
@@ -1,4 +1,67 @@
 #include "CourseDashboard.hpp"
 #include <stdexcept>
+#include <iostream>
 
 CourseDashboard::CourseDashboard() {}
+
+void CourseDashboard::showAll()
+{
+    for (auto & user : users_)
+    {
+        retriveUser(user);
+    }
+}
+
+void CourseDashboard::createUser(const User & user)
+{
+    users_.emplace_back(user);
+}
+
+void CourseDashboard::deleteUser(const User & user)
+{   
+    users_.erase(users_.begin());
```

Comment:
this always remove the first user. It's not a correct behavior.

Reply:
Fixed, as discussed today on course it is removing users by nick.